verification/certifyKey_test.go

```diff
@@ -14,19 +15,43 @@ import (
 	"github.com/zmap/zlint/v3/lint"
 )
 
-// This file is used to test the certify key command by using a simulator
+// This file is used to test the certify key command by using a simulator/emulator
+
+func GetTestTarget_CertifyKey() ([]TestDPEInstance, error) {
+
+	// Added dummy support for emulator
+	support_needed := []string{"AutoInit", "X509"}
+
+	var instances []TestDPEInstance
+
+	if testTargetType == EMULATOR {
+		return GetEmulatorTarget(support_needed, instances)
+
+	} else if testTargetType == SIMULATOR {
+		instances = []TestDPEInstance{
+			// No extra options besides AutoInit.
+			&DpeSimulator{exe_path: *socket_exe, supports: Support{AutoInit: true, X509: true}},
+			// Supports AutoInit and simulation contexts.
+			&DpeSimulator{exe_path: *socket_exe, supports: Support{AutoInit: true, Simulation: true, X509: true}},
+		}
+		for _, instance := range instances {
+			instance.SetLocality(DPE_SIMULATOR_AUTO_INIT_LOCALITY)
+		}
+		return instances, nil
+	}
+
+	return nil, errors.New("Error in creating dpe instances - supported feature is not enabled")
+}
 
 func TestCertifyKey(t *testing.T) {
-	simulators := []TestDPEInstance{
-		// No extra options besides AutoInit.
-		&DpeSimulator{exe_path: *sim_exe, supports: Support{AutoInit: true, X509: true}},
-		// Supports AutoInit and simulation contexts.
-		&DpeSimulator{exe_path: *sim_exe, supports: Support{AutoInit: true, Simulation: true, X509: true}},
+
+	instances, err := GetTestTarget_CertifyKey()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
If we error out because the requested `Support` isn't supported by the target, we should print a warning and then skip the test (instead of a fatal error).